.github/workflows/scripts/install-and-build-with-sdk.sh

```diff
@@ -112,11 +118,11 @@ find_latest_swift_version() {
             | .checksum
         ')
 
-        if [[ -z "$static_checksum" ]]; then
-            fatal "No static SDK checksum found for Swift $latest_version"
+        if [[ -z "$static_linux_checksum" ]]; then
+            fatal "No Static Linux Swift SDK checksum found for Swift $latest_version"
         fi
 
-        log "Found static SDK checksum: ${STATIC_SDK_CHECKSUM:0:12}..."
+        log "Found Static Linux Swift SDK checksum: ${STATIC_LINUX_SDK_CHECKSUM:0:12}..."
```

Comment:
Oops I made a mistake in the original script when refactoring, could you change this to the lowercase `static_linux_checksum`?

Reply:
Just there since it should be the local `static_linux_checksum` we just found and `STATIC_LINUX_SDK_CHECKSUM` isn't defined yet.

Reply:
Makes sense, that's addressed now!